ui/partials/perf/alertsctrl.html

```diff
@@ -115,10 +115,18 @@
     </div>
 
     <table class="table table-compact compare-table">
-      <tr ng-repeat="alert in alertSummary.alerts | orderBy: 'title'" ng-show="alert.visible">
+      <tr ng-repeat="alert in alertSummary.alerts | orderBy: ['-starred', 'title']" ng-show="alert.visible">
         <td class="alert-checkbox">
           <input type="checkbox" ng-disabled="!user.is_staff" ng-model="alert.selected" ng-change="alertSelected(alertSummary)"/>
         </td>
+        <td class="alert-labels">
+          <a title="Star the alert"
+             ng-class="{'fa fa-star visible': alert.starred, 'fa fa-star-o': !alert.starred}"
+             ng-disabled="!user.is_staff"
+             ng-hide="!user.is_staff && !alert.starred"
```

Comment:
I don't think we want to hide this data under any circumstances, make it read-only in the case that a user is a sheriff, sure.